src/sql/Dockerfile

```diff
@@ -57,5 +57,5 @@ ENTRYPOINT su - postgres -c "/usr/lib/postgresql/8.4/bin/postgres \
   "
 
 RUN pg_ctlcluster 8.4 main start && \
-    echo "create user gazadmin IN ROLE gazetteer_admin password 'gazadmin'" | \
+    echo "create user gazadmin IN ROLE gazetteer_admin, gazetteer_dba password 'gazadmin'" | \
```

Comment:
Shouldn't we keep plugin admin and dba functions separate?

Reply:
> Shouldn't we keep plugin admin and dba functions separate?

Thanks for the review I have done a review of roles (https://github.com/linz/gazetteer/issues/141) and will update this PR

Reply:
I have added all gaz roles to the docker db image.